class/defaults.yml

```
@@ -1,3 +1,7 @@
 parameters:
   openshift4_scheduling:
-    namespace: syn-openshift4-scheduling
+    namespace: openshift-kube-scheduler
+    schedulers:
+      cluster:
+        mastersSchedulable: false
+        defaultNodeSelector: node-role.kubernetes.io/app=
```

```suggestion
        defaultNodeSelector: node-role.kubernetes.io/worker=
```

In the sense of open source, `worker` is the value that is supposed to always work. Differentiating between `app` and `infra` is specific to VSHN.

---

Conclusion after discussing this with @tobru.

One principle of Syn is to provide sensible defaults. In this case, VSHN considers the use of infrastructure nodes a sensible default. Thus the defaults of the components shall reflect this.

However, this needs to be documented.